keyboards/ducky/one2mini/keymaps/ansi/keymap.c

```diff
@@ -0,0 +1,84 @@
+/* Copyright 2019 /u/KeepItUnder
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// LAYERS
+enum Layer {
+    _QWERTY = 0,        // Standard QWERTY layer
+    _FUNCTION,          // Function key layer
+    _COLOUR             // RGB key layer
+};
+#define _QW _QWERTY
+#define _FN _FUNCTION
+#define _CLR _COLOUR
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+  DK_SW1,
+  DK_SW2,
+  DK_SW3,
+  DK_SW4,
+  QMKBEST = SAFE_RANGE,
+  QMKURL
+};
```

Comment:
These are not implemented so can be removed, but at the least they need to follow the general pattern for custom keycodes.
```suggestion
// Defines the keycodes used by our macros in process_record_user
enum custom_keycodes {
  DK_SW1 = SAFE_RANGE,
  DK_SW2,
  DK_SW3,
  DK_SW4,
};
```